src/main/java/sirius/web/security/UserContext.java

```diff
@@ -208,21 +208,29 @@ private void bindScopeToRequest(WebContext ctx) {
      * Loads the current user from the given web context.
      */
     private void bindUserToRequest(WebContext ctx) {
-        UserManager manager = getUserManager();
         UserInfo user;
```

Comment:
then kill this uninitialized variable and call installUserInfoContext in each branch